docs/advanced-topics/dockstore-cli/local-cromwell-config.rst

```diff
@@ -0,0 +1,52 @@
+Using a Cromwell config with the Dockstore CLI
+==============================================
+
+Why set up a Cromwell configuration file?
+-----------------------------------------
+
+It is not necessary to set up a configuration file to use Cromwell or the Dockstore CLI, but advanced users may find some settings they may want to change. Additionally, setting up a Cromwell configuration file is the best way to avoid :ref:`Docker lockups and sigkilled tasks <cromwell-docker-lockup>` from happening when working with scattered tasks. Because avoiding those lockups is one of the more common use cases for setting up a Cromwell configuration file, we will focus on that specific use case here, but you can apply this to modify other Cromwell settings as needed.
+
+To help keep scattered tasks from taking too many local resources, you can set up a Cromwell configuration file that sets a concurrent-job-limit. Of course, there is a tradeoff: If you set a concurrent-job-limit, tests involving scattered tasks will execute slower as less instances of a scattered task will run in parallel. That being said, we still recommend setting this value when running on a local machine, as it makes Cromwell much more stable.
+
+1. Download `this template file <https://github.com/broadinstitute/cromwell/blob/develop/cromwell.example.backends/cromwell.examples.conf>`__ and name it ``.cromwell.conf`` before placing it in your Users folder, ie ``/Users/ash/``, which is the same folder where ``.bashrc`` and other configuration files tend to go. If you are on Mac OS, you may get a notification about naming the file like this -- this is normal; Mac OS normally hides files that have a period at the start of the name (called "dot files") to prevent people from deleting important operating system files by accident. 
```

Comment:
> it in your Users folder, ie ``/Users/ash/``

1. I'd call it the home directory
2. Might want to give a Linux example as well, e.g., /home/ash.